### DISCUSSION TITLE
AospExtended 12.1.x Sync Error

### DISCUSSION THREAD

**Discussion by romgharti** [ROM-builders/temporary#9130] [Q&A]
https://cirrus-ci.com/task/5981624285265920

**Comment by Apon77** [comment]
> https://cirrus-ci.com/task/5981624285265920

try locally first

**Reply by romgharti** [reply]
> > https://cirrus-ci.com/task/5981624285265920
> 
> try locally first
i tried sync locally and fine sync

**Comment by rushiranpise** [comment]
https://cirrus-ci.com/task/4982890508320768

ig src side
